From 809e38acf1e5bbd2149f5e443d8e08c4b24dd731 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 14 Sep 2017 19:54:24 +0800
Subject: [PATCH] FIXUP: sched: fix build for non-SMP target

Fix merge error. kernel/sched/energy.c should not build on non-SMP.

Fixes: f9ae5d202b39 ("Merge branch 'linux-linaro-lsk-v4.4-android' of git://git.linaro.org/kernel/linux-linaro-stable.git")
Change-Id: I5dba90ec363c02e489a48d76bb97f9f9d62827cd
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 kernel/sched/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/sched/Makefile b/kernel/sched/Makefile
index 307c931272de..ca0d94096170 100644
--- a/kernel/sched/Makefile
+++ b/kernel/sched/Makefile
@@ -12,7 +12,7 @@ CFLAGS_core.o := $(PROFILING) -fno-omit-frame-pointer
 endif
 
 obj-y += core.o loadavg.o clock.o cputime.o
-obj-y += idle_task.o fair.o rt.o deadline.o stop_task.o energy.o
+obj-y += idle_task.o fair.o rt.o deadline.o stop_task.o
 obj-y += wait.o completion.o idle.o
 obj-$(CONFIG_SMP) += cpupri.o cpudeadline.o energy.o
 obj-$(CONFIG_SCHED_WALT) += walt.o
-- 
2.35.3

